tests/nnstreamer_filter_tensorflow2_lite/runTest.sh

```diff
@@ -69,6 +69,7 @@ fi
 PATH_TO_MODEL="../test_models/models/mobilenet_v1_1.0_224_quant.tflite"
 PATH_TO_LABEL="../test_models/labels/labels.txt"
 PATH_TO_IMAGE="../test_models/data/orange.png"
+PATH_TO_OPENCL="/usr/lib/libOpenCL.so"
```

Comment:
I think it possibly causes false negative cases when checking the opencl installation.
For example, my desktop has opencl library located in `/usr/lib/x86_64-linux-gnu/libOpenCL.so`

Reply:
How about using an environment variable set from a build script or other ways.

Reply:
```
$ ldconfig -p | grep "libOpenCL.so " | awk '{print $4}'
/usr/local/cuda-10.2/targets/x86_64-linux/lib/libOpenCL.so
```

I am not sure that this works on Tizen.